code/game/gamemodes/traitor/traitor.dm

```diff
@@ -154,3 +189,13 @@
 					<b>The code responses were:</b> <span class='danger'>[responses]</span><br><br>"
 
 		return text.Join("")
+
+/datum/game_mode/proc/reveal_traitor_objectives()
```

Comment:
This one is quite misnamed now. Also, having a `game_mode` scope proc here in this file feels weird.

Reply:
I renamed it to `clean_up_traitor_list` which might (?) be better than before.
The reason it is a `game_mode` scope is because trifecta also uses this exact same code, and this is the only common scope as trifecta isn't a derivative of `game_mode/traitor`